recipe/recipe.yaml

```diff
@@ -87,6 +87,9 @@ source:
     # 2024/08 hmaarrfk
     # known flaky test https://github.com/pytorch/vision/blob/9e78fe29e0851b10eb8fba0b88cc521ad67cf322/test/test_image.py#L840
     - patches/0003-Skip-OSS-CI-in-conda-forge-as-well.patch
+    # Point users to use the conda-forge package for the extra codecs (heif and avif)
+    # https://github.com/conda-forge/torchvision-feedstock/pull/115
+    - patches/0004-Point-users-to-the-conda-package-for-extra-codecs.patch
```

Comment:
@hmaarrfk, it seems you never committed that patch?

Reply:
i guess i lost it.

Reply:
well, are you OK with adding heiv/avif support by default then?

Reply:
I just forget where things stand. 

I would prefer to just patch the recommendations since these recommendations are likely incompatible with our stuff

Reply:
good point. i likely won't have time to provide a patch anytime soon.

Reply:
feel free to hard depend if you want.